include/rocksdb/sst_file_manager.h

```diff
@@ -64,17 +65,12 @@ class SstFileManager {
 // @param info_log: If not nullptr, info_log will be used to log errors.
 //
 // == Deletion rate limiting specific arguments ==
-// @param trash_dir: Path to the directory where deleted files will be moved
-//    to be deleted in a background thread while applying rate limiting. If this
-//    directory doesn't exist, it will be created. This directory should not be
-//    used by any other process or any other SstFileManager, Set to "" to
-//    disable deletion rate limiting.
+// @param trash_dir: Deprecated, this argument have no effect
```

Comment:
Even if it is deprecated, we need to add those files into the delete schedule queue. Otherwise, these file will never be removed after they upgrade the RocksDB to the new version without manually clean up existing files there. This is not going to be a good experience to them.

Reply:
Sounds good, this is what we do right now, we delete files in trash_dir